banzai/mosaic.py

```diff
@@ -18,13 +18,19 @@ def do_stage(self, image):
             nx, ny = get_mosaic_size(image, image.get_n_amps())
             mosaiced_data = np.zeros((ny, nx), dtype=np.float32)
             mosaiced_bpm = np.zeros((ny, nx), dtype=np.uint8)
+            x_detsec_limits, y_detsec_limits = get_detsec_limits(image, image.get_n_amps())
+            xmin = min(x_detsec_limits) - 1
+            ymin = min(y_detsec_limits) - 1
             for i in range(image.get_n_amps()):
+                ccdsum = image.extension_headers[i].get('CCDSUM', image.ccdsum)
```

Comment:
It is understood that all extensions wil have the same binning. Do you want to assert this here?

Reply:
This should cover the case when they are different. For the moment, I think this makes sense.